nautobot/utilities/testing/api.py

```diff
@@ -146,15 +146,19 @@ def test_get_object(self):
 
             # If opt-in fields are supported on this model, make sure they can be opted into
 
-            custom_fields_registry = registry["model_features"]["custom_fields"]
-            # computed fields and custom fields use the same registry
-            cf_supported = self.model._meta.model_name in custom_fields_registry.get(self.model._meta.app_label, {})
+            registry.custom_fields_registry = registry["model_features"]["custom_fields"]
+            # computed fields and custom fields use the same registry.registry
+            cf_supported = self.model._meta.model_name in registry.custom_fields_registry.get(
+                self.model._meta.app_label, {}
+            )
             if cf_supported:  # custom_fields is not an opt-in field, it should always be present if supported
                 self.assertIn("custom_fields", response.data)
                 self.assertIsInstance(response.data["custom_fields"], dict)
 
-            relationships_registry = registry["model_features"]["relationships"]
-            rel_supported = self.model._meta.model_name in relationships_registry.get(self.model._meta.app_label, {})
+            registry.relationships_registry = registry["model_features"]["relationships"]
+            rel_supported = self.model._meta.model_name in registry.relationships_registry.get(
+                self.model._meta.app_label, {}
+            )
```

Comment:
This looks weird - why are we assigning into the `registry` module?

Reply:
Looks like the regex (or my brain) got confused here